airbyte-workers/src/main/java/io/airbyte/workers/worker_run/TemporalWorkerRunFactory.java

```diff
@@ -100,4 +123,15 @@ private static TemporalJobType toTemporalJobType(final ConfigType jobType) {
     return new OutputAndStatus<>(status, new JobOutput().withSync(response.getOutput().orElse(null)));
   }
 
+  private OutputAndStatus<JobOutput> toOutputAndStatusConnector() {
+    // Since we are async we technically can't fail
```

Comment:
This feels really weird here but I can't think of a way of doing it better. Probably fine for now.